portable/GCC/ARM7_AT91FR40008/portISR.c

```diff
@@ -163,50 +163,50 @@ void vPortYieldProcessor( void )
+            "BX     R14" );               /* Return back to thumb.                    */
     }
 
     void vPortEnableInterruptsFromThumb( void )
     {
         asm volatile (
-            "STMDB  SP!, {R0}       \n\t"   /* Push R0.                                 */
-            "MRS    R0, CPSR        \n\t"   /* Get CPSR.                                */
-            "BIC    R0, R0, #0xC0   \n\t"   /* Enable IRQ, FIQ.                         */
-            "MSR    CPSR, R0        \n\t"   /* Write back modified value.               */
-            "LDMIA  SP!, {R0}       \n\t"   /* Pop R0.                                  */
-            "BX     R14" );                 /* Return back to thumb.                    */
+            "STMDB  SP!, {R0}       \n\t" /* Push R0.                                 */
+            "MRS    R0, CPSR        \n\t" /* Get CPSR.                                */
+            "BIC    R0, R0, #0xC0   \n\t" /* Enable IRQ, FIQ.                         */
+            "MSR    CPSR, R0        \n\t" /* Write back modified value.               */
+            "LDMIA  SP!, {R0}       \n\t" /* Pop R0.                                  */
+            "BX     R14" );               /* Return back to thumb.                    */
     }
 
 #endif /* THUMB_INTERWORK */
 
 /* The code generated by the GCC compiler uses the stack in different ways at
-different optimisation levels.  The interrupt flags can therefore not always
-be saved to the stack.  Instead the critical section nesting level is stored
-in a variable, which is then saved as part of the stack context. */
+ * different optimisation levels.  The interrupt flags can therefore not always
+ * be saved to the stack.  Instead the critical section nesting level is stored
+ * in a variable, which is then saved as part of the stack context. */
 void vPortEnterCritical( void )
 {
     /* Disable interrupts as per portDISABLE_INTERRUPTS();                          */
     asm volatile (
-        "STMDB  SP!, {R0}           \n\t"   /* Push R0.                             */
-        "MRS    R0, CPSR            \n\t"   /* Get CPSR.                            */
-        "ORR    R0, R0, #0xC0       \n\t"   /* Disable IRQ, FIQ.                    */
-        "MSR    CPSR, R0            \n\t"   /* Write back modified value.           */
-        "LDMIA  SP!, {R0}" );               /* Pop R0.                              */
+        "STMDB  SP!, {R0}           \n\t" /* Push R0.                             */
+        "MRS    R0, CPSR            \n\t" /* Get CPSR.                            */
+        "ORR    R0, R0, #0xC0       \n\t" /* Disable IRQ, FIQ.                    */
+        "MSR    CPSR, R0            \n\t" /* Write back modified value.           */
+        "LDMIA  SP!, {R0}" );             /* Pop R0.                              */
 
     /* Now interrupts are disabled ulCriticalNesting can be accessed
```

Comment:
Consider changing to "Now that interrupts are disabled.." or something similar

Reply:
Keeping this PR as exclusively typos and formatting changes, not going to address any other content than this, even though I can recognize that you are correct here.

Reply:
Fixed